src/components/panel/panel.spec.js

```diff
@@ -1475,6 +1696,29 @@ describe('$mdPanel', function() {
   }
 
   /**
+   * Attaches the panel. If a config value is passed, creates a new panelRef
+   * using $mdPanel.open(config); Otherwise, called open on the panelRef,
+   * assuming one has already been created.
+   * @param {!Object=} opt_config
+   */
+  function attachPanel(opt_config) {
+    var attachPromise; 
+    if (opt_config) {
+      panelRef = $mdPanel.create(opt_config);
```

Comment:
This doesn't work. `$mdPanel.create` returns a promise that resolves to panelRef, not panelRef.